build.sbt

```diff
@@ -93,6 +93,10 @@ lazy val core = crossProject(JSPlatform, JVMPlatform, NativePlatform)
 lazy val coreJVM = core.jvm
   .settings(dottySettings)
   .settings(replSettings)
+  .settings(
+    libraryDependencies +=
+      "org.scala-lang" % "scala-reflect" % scalaVersion.value
```

Comment:
I see this is only used on the jvm but I think we need to be careful if we ever consider GraalVM native-image to be a viable "platform" for ZIO as it supports _very limited_ reflection features.

Reply:
The only thing we need is `TypeTag` (or `ClassTag` as a poor man's substitute) which is a compile-time feature.